### PR TITLE
Assignment5 and assignment6

### PR DESCRIPTION
**Pull request by vdongre304**

### CODE REVIEW

**Review by Jestinxavier — APPROVED** [review]

**Review by yogesh545 — APPROVED** [review]

File: vinitD/html/asgnmnt6/formcontinue.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                    </td>
+                </tr>
+            </table>
+        <br><br>
+        <div>
+            <table>
+                <tr>
+                    <td>Highest Qualification :</td>
+                    <td>
+            <select>
```
Comment (KomalShinde813) [inline vinitD/html/asgnmnt6/formcontinue.html]: check indentation for select block

File: vinitD/html/asgnmnt6/formcontinue.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+                </tr>
+            </table>
+        </div>
+    </div>
+        <br><br>
+        <div>
+            <h2 class="h2">Resume</h2>
+            <table >
+                <div class="x">
+                <tr>
```
Comment (KomalShinde813) [inline vinitD/html/asgnmnt6/formcontinue.html]: check indentation here

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

File: vinitD/html/asgnmnt6/formcontinue.html [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+            <table>
+                <tr>
+                    <td>Skill Set :</td>
+                    <td>
+                        <input type="checkbox" name="Skill Set" />HTML <input type="checkbox" name="Skill Set" />Java Script
+                        <input type="checkbox" name="Skill Set" />Java <input type="checkbox" name="Skill Set" />Oracle
+                    </td>
+                </tr>
+            </table>
+        <br><br>
```
Comment (KomalShinde813) [inline vinitD/html/asgnmnt6/formcontinue.html]: this line should be exact below table tag

File: vinitD/html/asgnmnt6/formcontinue.html [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+        <div>
```
Comment (KomalShinde813) [inline vinitD/html/asgnmnt6/formcontinue.html]: check this indentation according to above comment

File: vinitD/html/asgnmnt6/formcontinue.html [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+                <tr>
+                    <td>Upload Resume :</td>
+                    <td>
+                        <button onclick="document.getElementById('file').click()">Browse...</button>No file selected
+                        <input type="file" id="file" name="files" style="display:none;"/>
+                    </td>
+                </tr>
+            </div>
+            </table>
+        <br><br>
```
Comment (KomalShinde813) [inline vinitD/html/asgnmnt6/formcontinue.html]: check indentation here also

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

File: vinitD/html/asgnmnt6/formcontinue.html [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+                            <option value="Bachelor in Engg.">Bachelor In Engg.</option>
+                            <option value="Masters in Computer Science">Master In Computer Science</option>
+                            <option value="Masters in Engg.">Master In Engg.</option>
+                            <option value="Other">Other</option>
+                        </select>
+                    </td>
+                </tr>
+            </table>
+            <br><br>
+        <div>
```
Comment (KomalShinde813) [inline vinitD/html/asgnmnt6/formcontinue.html]: check indentation from this line div should be exact below of br tag

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

**Review by KomalShinde813 — APPROVED** [review]

**Pull request merged**